src/docs/arch-metadata.dox

```diff
@@ -1,19 +1,78 @@
 /*! @arch_page arch-metadata Metadata
 
-Metadata in WiredTiger is stored as a table in the \c "WiredTiger.wt" file.
-The table's key is a \c uri string, and the value is its complete
-configuration string.  The configuration itself is a list of key/value
-pairs in string form.  The configuration's keys are dependent on the
-type of \c uri.  Thus, a metadata entry with a \c uri key beginning with
-\c "table:" will be a configuration string
-having configuration entries like \c key_format and \c value_format to
-describe the data encoding for the uri.  A metadata key beginning with
+Metadata in WiredTiger captures important information about the users database.
+Metadata includes tracking essential information such as the files and tables
+present in the database, their associated configurations, as well as the latest checkpoints of
+each those files.
```

Comment:
Maybe add another sentence like: The checkpoint information tells WiredTiger where to find the root page and all other parts of the tree when accessing the file.

Reply:
That's a great idea, totally missed that detail. I've updated the introduction with your suggestion: d8ca389c620422ba159f8e51e0450a3990a3869f